Makefile

```diff
@@ -61,41 +69,37 @@ help:
 	@echo "- ci                 Create the python virtual environment and install requirements based on the Pipfile.lock"
 	@echo -e " \033[1mFORMATING, LINTING AND TESTING TOOLS TARGETS\033[0m "
 	@echo "- format             Format the python source code"
-	@echo "- ci-check-format    Format the python source code and check if any files has changed. This is meant to be used by the CI."
```

Comment:
The `ci-check-format` should stay ! It improves the code quality by forcing the PR to be formatted.